<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CurrentListControlWidget</class>
 <widget class="QWidget" name="CurrentListControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">QWidget#frame{
background-color:#ECAD9E  ;
color:white;
border:none;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,1">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="labelName">
            <property name="font">
             <font>
              <family>Mistral</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>标签名</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item>
           <widget class="QLabel" name="Detail">
            <property name="font">
             <font>
              <family>MingLiU_HKSCS-ExtB</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>详情信息</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textEdit"/>
          </item>
          <item>
           <widget class="QLabel" name="Type">
            <property name="font">
             <font>
              <family>Mistral</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>类型</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox"/>
          </item>
          <item>
           <widget class="QPushButton" name="AddTask">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>添加任务</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>Monotype Corsiva</family>
              <pointsize>14</pointsize>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string>对应任务星级数</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="StarSpinBox">
            <property name="suffix">
             <string>颗</string>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>1.000000000000000</double>
            </property>
            <property name="maximum">
             <double>5.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.500000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="MyMaterialFlatButton" name="Save">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>保存</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="MyMaterialFlatButton" name="pushButton">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>删除</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="MyMaterialFlatButton" name="Query">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>？</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyMaterialFlatButton</class>
   <extends>QPushButton</extends>
   <header location="global">mymaterialflatbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
